api/api/src/test/scala/uk/ac/wellcome/platform/api/models/AggregationResultsTest.scala

```diff
@@ -0,0 +1,21 @@
+package uk.ac.wellcome.platform.api.models
+
+import org.scalatest.{FunSpec, Matchers}
+
+class AggregationResultsTest extends FunSpec with Matchers {
+  it("destructures a single aggregation result") {
+    // val searchResponse: com.sksamuel.elastic4s.requests.searches.SearchResponse
+
+    // This mimics the behaviour of searchResponse.aggregationsAsMap
+    val singleAggregationsMap = Map("workType" -> Map.empty)
+
+    //  And this mimics the searchResponse.aggregationsAsString
+    val responseString =
+      """
+        |{"workType":{"doc_count_error_upper_bound":0,"sum_other_doc_count":0,"buckets":[{"key":"b","doc_count":2},{"key":"a","doc_count":1},{"key":"m","doc_count":1}]}}
+        |""".stripMargin
```

Comment:
Maybe parse this to a circe `Json` object for the test, so that it is not dependent on formatting, and can therefore use indentation when defining it? Should hopefully result in nicer error message upon failure too.

See here if not sure what I mean: https://github.com/wellcometrust/catalogue/pull/146/files#diff-1cd03f8075a9a86e5d6cb75387924cc6R129-R138

Reply:
I've reformatted the string, but the point of this test it to check that we can parse an Elasticsearch search JSON string into our case classes, as Elastic4S doesn't model those out, so I've left it as a string.

There was an error in the test that probably didn't help explain this which has been fixed.